Package.swift

```diff
@@ -13,7 +13,7 @@ let package = Package(
         .library(name: "PostgresKit", targets: ["PostgresKit"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/vapor/postgres-nio.git", from: "1.14.2"),
+        .package(url: "https://github.com/vapor/postgres-nio.git", exact: "1.17.0"),
```

Comment:
I would've used an upper-bound of 1.17.0 instead of "exact".
Someone might want to have their PostgresNIO pinned to a lower version (believe me, i've seen those, mostly as a mistake, but anyway)

Reply:
The other issue this might cause is people pinning to old versions for the error stuff as well